### ISSUE TITLE
Python 2.7.x compatibility?

### ISSUE THREAD

**Issue by evandrix** [HashCode55/Sniper#1]
any plans to port this to be Python2 compatible?

**Comment by rombiee** [comment]
Yeah, definitely. But eventually, I'd want Sniper to be a standalone application without any dependency. Probably packaging it using py2app/py2exe or something. 

Currently, the project is structured as a `pip` package and I think py2app/py2exe doesn't expect this kind of structure. Although, making it Python 2.x compatible for now won't be a tough task to do. 

My priority right now is to implement the features listed in [TODO](https://github.com/HashCode55/Sniper#todo) and make the tool more appealing and convenient to use.
